packages/@okta/vuepress-theme-prose/assets/css/components/_guidesOverview.scss

```diff
@@ -0,0 +1,82 @@
+.guides {
+  
+    &--features {
+        display: flex;
+        width: 64%;
+        max-width: 64%;
+    }
+
+    &--featured {
+    transition: box-shadow 200ms linear;
+    border: 0;
+    background: white;
+    box-shadow: 0 3px 6px rgba(47, 63, 74, 0.2);
+    text-decoration: none !important;
+    cursor: pointer;
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    width: 50%;
+    border-radius: 3px;
+  
+    &:first-child {
+      margin-right: 40px;
+    }
+  
+    &:hover {
+      box-shadow: 0 6px 24px rgba(47, 63, 74, 0.2);
+      color: inherit;
+      text-decoration: none;
+    }
+  
+    /* reduce icons to 24px */
+    .icon[class*='-32'] {
```

Comment:
Not a problem today, but I think this would break if a guide such as [OIN - Build a Single Sign-On (SSO) integration](https://developer.okta.com/docs/guides/build-sso-integration/) was featured. Not all icons include `-32`. The StackSelector component has similar (newer) CSS.

Reply:
ah. I'll update the css in next commit!